templates/Edit.mediawiki

```diff
@@ -99,7 +99,11 @@
             }}
 
             <textarea name="content" style="width: 100%;" rows=30>{{page}}</textarea>
+            {{#if:{{{has_summary_text|}}}|
+            <input type="text" name="summary" maxlength="500" value="{{summary_text}}" style="width: 100%;" />
+            |
             <input type="text" name="summary" maxlength="500" placeholder="Edit Summary (Optional)..." style="width: 100%;" />
+            }}
```

Comment:
Why not just do only `value="{{summary_text}}"` ? (instead of copying the whole line).

Now the new line doesn't have a placeholder, so if someone empties the textfield, he doesn't see that anymore. Feels a bit weird, but you might have had a good reason for it, hence me asking :D

Reply:
Well I tested it out and setting the value at all, even to an empty string, overrides the placeholder text. I can add the placeholder text back (that was an oversight), but that's why we need the if-statement with the different line.

Reply:
For some reason I had it in my head that you couldn't have both a `placeholder` and `value` set at the same time. I'll make the change really quick :)